docs/user_guide/SIMP_Administration/Classification_and_Data/SIMP_Scenarios.inc

```diff
@@ -0,0 +1,35 @@
+.. _simp scenarios:
+
+SIMP Scenarios
+--------------
+
+SIMP scenarios are groups of classes, settings, and simp_options that ensure the
+system is compliant and secure.
+
+There are currently three SIMP scenarios:
```

Comment:
Created at ticket https://simp-project.atlassian.net/browse/SIMP-3287 to track all of this for the future.